src/haplotypes/model.rs

```diff
@@ -127,37 +132,16 @@ impl Likelihood {
         // TODO compute likelihood based on Varlociraptor VAFs.
         // Let us postpone this until we have a working version with kallisto only.
         let variant_matrix: Vec<BitVec> = data.haplotype_variants.values().cloned().collect();
-        let likelihood = Prob(1.0);
-        for (variant, bv) in data.haplotype_variants.iter() {
+        let mut likelihood = Prob(1.0);
+        for (bv, (_, afd)) in variant_matrix.iter().zip(data.haplotype_calls.iter()) {
             let variant_sum = 0.0;
             for (i, fraction) in event.iter().enumerate() {
                 if bv[i.try_into().unwrap()] == true {
```

Comment:
`== true` is never needed in an if condition, since it already checks for the condition to be true anyway.